src/api/grainConfig.test.js

```diff
@@ -3,122 +3,237 @@
 import {parser, toDistributionPolicy, type GrainConfig} from "./grainConfig";
 import {type DistributionPolicy} from "../core/ledger/applyDistributions";
 import {toDiscount} from "../core/ledger/policies/recent";
+import {type Uuid, random as randomUuid} from "../util/uuid";
 import {fromInteger as toNonnegativeGrain} from "../core/ledger/nonnegativeGrain";
+import {type BalancedPolicy} from "../core/ledger/policies/balanced";
+import {type ImmediatePolicy} from "../core/ledger/policies/immediate";
+import {type RecentPolicy} from "../core/ledger/policies/recent";
+import {type SpecialPolicy} from "../core/ledger/policies/special";
+
+const balanced = (budget: number): BalancedPolicy => ({
+  policyType: "BALANCED",
+  budget: toNonnegativeGrain(budget),
+});
+const immediate = (budget: number): ImmediatePolicy => ({
+  policyType: "IMMEDIATE",
+  budget: toNonnegativeGrain(budget),
+});
+const recent = (budget: number, discount: number): RecentPolicy => ({
+  policyType: "RECENT",
+  budget: toNonnegativeGrain(budget),
+  discount: toDiscount(discount),
+});
+const special = (
+  budget: number,
+  memo: string,
+  recipient: Uuid
+): SpecialPolicy => ({
+  policyType: "SPECIAL",
+  budget: toNonnegativeGrain(budget),
+  memo,
+  recipient,
+});
 
 describe("api/grainConfig", () => {
   describe("parser", () => {
-    it("does not throw with no params", () => {
-      expect(parser.parseOrThrow({})).toEqual({});
-    });
```

Comment:
Let's keep this test? It should still be fine with no params.

Reply:
Done